backend/docker-compose.yml

```diff
@@ -25,7 +24,7 @@ services:
             - '-c'
             - 'docker-entrypoint.sh --requirepass "$$REDIS_PASSWORD"'
         ports:
-            - 127.0.0.1:${REDIS_PORT}:6379
+            - 127.0.0.1:${REDIS_PORT}:${REDIS_PORT}
```

Comment:
no, no, no
the last port is static, cannot be changed

Reply:
127.0.0.1:${REDIS_PORT}:6379

Reply:
done